docs/explanation/build-plans.rst

```diff
@@ -0,0 +1,56 @@
+Build plans
```

Comment:
```suggestion
Build Plans
```

(Title case)

Reply:
Should it be sentence case?
https://docs.ubuntu.com/styleguide/en#headings

Reply:
Ahh, thank you. *Adjusts his own documentation he's writing*